docs/release-notes/5.7.42-46.md

```diff
@@ -0,0 +1,47 @@
+# Percona Server for MySQL 5.7.42-46 (2023-06-01)
+
+<style>
+    table {
+        width=50%;
+        font-family: Poppins;
+    }
+    table td {
+        border: 0px;
+        padding: 8px;
+    }
+</style>
+
+| Release date | June 01, 2023 |
+|---|---|
+| Install instructions | [Install Percona Server for MySQL](https://www.percona.com/doc/percona-server/5.7/installation.html) |
+
+[Percona Server for MySQL](https://www.percona.com/software/mysql-database/percona-server) 5.7.42-46
+includes all the features and bug fixes available in [MySQL 5.7.42 Community Edition](https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-42.html) in addition to enterprise-grade features developed by Percona.
+
+Percona Server for MySQL is a freely available, fully compatible, enhanced, and open source drop-in replacement for any MySQL database. It provides superior and optimized performance, greater scalability and availability, enhanced backups, increased visibility, and instrumentation .
+
+Percona Server for MySQL is trusted by thousands of enterprises to provide
+better performance and concurrency for their most demanding workloads. 
+
+## Release highlights
+
+Improvements and bug fixes provided by Oracle for MySQL 5.7.42 and included in Percona Server for MySQL are the following:
+
+* In InnoDB, online DDL operations are prevented from accessing out-of-bounds memory
+  
+Find the full list of bug fixes and changes in the [MySQL 5.7.42 Release Notes](https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-42.html).
+
+## Bug fixes
+
+* [PS-8779](https://jira.percona.com/browse/PS-8779): The `mysqldump` client utility ignored `--set-gtid-purged=OFF`.
```

Comment:
Wrong Jira ticket here
the correct one is https://jira.percona.com/browse/PS-8776

Reply:
Fixed